src/pages/send-tokens/hooks/use-send-form-validation.ts

```diff
@@ -94,6 +101,9 @@ export const useSendFormValidation = ({ setAssetError }: UseSendFormValidationAr
     } else {
       setAssetError(SendFormErrorMessages.MustSelectAsset);
     }
+    if (!isTxMemoValid(memo)) {
+      errors.memo = 'Memo must be within 34-byte limit';
```

Comment:
I feel like could equally say `Memo must be 34 characters or less.`

Also, should be enum

Reply:
It's not exactly the same, though. It's not 1-byte per char.

Reply:
right -- i think we should try to find a more friendly message, maybe something like `Memo is too long. Received ${memoBuff.length}, expected: 34.`

I just think the 34 bytes language is not really helpful imo. @markmhx @jasperjansz any suggestions?

Reply:
Generally 1-char is one byte, rather, but each one of these family emojis is 25 bytes 👨‍👩‍👧‍👦.  So 2 is already over the limit.